Back-End/controllers/registration_controllers/register_controllers.js

```diff
@@ -2,26 +2,29 @@ const connection = require('../../db');
 const bcrypt = require('bcrypt');
 
 const  register =(req,res)=>{
-const{name,adress,email,password,phone} =req.body
+const{name,adress,email,password,phone,confPassword} =req.body
   let role_id =0
 
     //setting the role for the registered account
-    if(req.params.role ==="instructor"){
+    if(req.params.role === "instructor"){
         role_id = 2
-     }else if(req.params.role ==="student"){
+     }else if(req.params.role === "student"){
         role_id = 3
      };
+     if(password!==confPassword){
+       return res.json("Passwords don't match..")
+     }
```

Comment:
If the passwords do not match it is supposed to be a front end validation rather than a backend one